[2024-12-02 06:47:47,266] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: kpi_calculation_pipeline.store_kpi 2024-11-23T01:05:00+00:00 [queued]>
[2024-12-02 06:47:47,316] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: kpi_calculation_pipeline.store_kpi 2024-11-23T01:05:00+00:00 [queued]>
[2024-12-02 06:47:47,317] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-12-02 06:47:47,319] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2024-12-02 06:47:47,319] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-12-02 06:47:47,418] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): store_kpi> on 2024-11-23T01:05:00+00:00
[2024-12-02 06:47:47,424] {standard_task_runner.py:52} INFO - Started process 232 to run task
[2024-12-02 06:47:47,429] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'kpi_calculation_pipeline', 'store_kpi', '2024-11-23T01:05:00+00:00', '--job-id', '402', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/data_injestion_dag/bazefield_kpi_calculation.py', '--cfg-path', '/tmp/tmpdk_2q7ii', '--error-file', '/tmp/tmpaclbt3b2']
[2024-12-02 06:47:47,432] {standard_task_runner.py:77} INFO - Job 402: Subtask store_kpi
[2024-12-02 06:47:47,607] {logging_mixin.py:104} INFO - Running <TaskInstance: kpi_calculation_pipeline.store_kpi 2024-11-23T01:05:00+00:00 [running]> on host 59507e5d9338
[2024-12-02 06:47:47,843] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=kpi_calculation_pipeline
AIRFLOW_CTX_TASK_ID=store_kpi
AIRFLOW_CTX_EXECUTION_DATE=2024-11-23T01:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-23T01:05:00+00:00
[2024-12-02 06:47:47,896] {base.py:69} INFO - Using connection to: id: postgres_default. Host: 10.197.48.9, Port: 5432, Schema: kpi_data, Login: rakshit, Password: ***, extra: {}
[2024-12-02 06:47:48,072] {logging_mixin.py:104} INFO - successfully connected
[2024-12-02 06:47:48,155] {python.py:151} INFO - Done. Returned value was: None
[2024-12-02 06:47:48,182] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=kpi_calculation_pipeline, task_id=store_kpi, execution_date=20241123T010500, start_date=20241202T064747, end_date=20241202T064748
[2024-12-02 06:47:48,327] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-02 06:47:48,346] {local_task_job.py:151} INFO - Task exited with return code 0
